security/cert_manager_operator/cert-manager-securing-routes.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-securing-routes"]
+= Securing routes with the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-securing-routes
+
+toc::[]
+
+In the {product-title}, the route API is extended to provide a configurable option to reference TLS certificates via secrets. With the xref:https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-networking-secured-route_release-notes[Creating a route with externally managed certificate] feature enabled, you can minimize errors from manual intervention, streamline the certificate management process, and enable the {product-title} router to promptly serve the referenced certificate.
+
+:FeatureName: Securing routes with the {cert-manager-operator}
+include::snippets/technology-preview.adoc[]
+
+include::modules/cert-manager-configuring-routes.adoc[leveloffset=+1]
+
+[id="additional-resources_cert-manager-securing-routes"]
+== Additional resources
```

Comment:
Per updated [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies):
```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```